articles/search/search-region-support.md

```diff
@@ -40,8 +40,8 @@ You can create an Azure AI Search resource in any of the following Azure public
 | Brazil South​​ ​ | ✅ | ✅ | |  |
 | Canada Central​​ | ✅ | ✅ | ✅ |  |
 | Canada East​​ ​ |  | ✅ | |  |
-| East US​ <sup>1</sup>| ✅ | ✅ | ✅ | <span class='red-x'>&#10060;</span> |
-| East US 2 ​ | ✅ | ✅ | ✅ | |
+| East US​ <sup>1</sup>| ✅ | ✅ | ✅ |  |
```

Comment:
the footnote and the red x are now on different regions, sending a mixed signal.  Is it East US or East US 2 that is capacity constrained?

Reply:
We enabled East US yesterday and disabled East US2 yesterday. I'm updating the tiers doc with these changes too.